modules/http/src/main/java/org/mule/module/http/internal/listener/grizzly/ExecutorPerServerAddressIOStrategy.java

```diff
@@ -29,6 +30,7 @@
  */
 public class ExecutorPerServerAddressIOStrategy extends AbstractIOStrategy
 {
+    protected final static String UNAVAILABLE_ATTRIBUTE = "__reject_request__";
```

Comment:
You could use `org.glassfish.grizzly.attributes.Attribute<Boolean>` here, but up to you.